### PR TITLE
ZENKO-4395: bump version and vault

### PR DESCRIPTION
**Pull request by ghost**
**What does this PR do, and why do we need it?**
Bumping Zenko version to update with new Vault version

**Which issue does this PR fix?**
Adds Vault caching PRs:
https://github.com/scality/vault2/pull/95
https://github.com/scality/vault2/pull/96
https://github.com/scality/vault2/pull/97
https://github.com/scality/vault2/pull/100
https://github.com/scality/vault2/pull/93
https://github.com/scality/vault2/pull/94

**Special notes for your reviewers**:

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1,4 +1,4 @@
-VERSION="2.5.7"
+VERSION="2.5.8"
```
Comment (ghost) [inline VERSION:1]: Let's try to have this version bump as the last commit, the one that will be tagged, if possible :slightly_smiling_face:

**Review by ghost — APPROVED** [review]

File: VERSION (hunk 1 above)
Comment (ghost) [reply VERSION:1]: oh ok!

**Review by anurag4DSB — APPROVED** [review]

**Comment by ghost** [comment]
ping

**Pull request merged**